projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator.ts

```diff
@@ -360,16 +360,21 @@ export function checkValueSelected(
 ): void {
   const attributeId = getAttributeId(attributeName, uiType);
   let valueId = `${attributeId}--${valueName}`;
+  cy.log('valueId: ' + valueId);
```

Comment:
In case you need it: Please make this message more specific (should be clear what the context is).

Reply:
Done!